popup.html

```diff
@@ -1,44 +1,33 @@
 ﻿<!DOCTYPE html>
 <html>
-  <head>
-    <link rel="stylesheet" href="popup.css">
-  </head>
-  <body>
-  
-    <div class="container">
-      <input type="text" id="searchField" placeholder="Search Asset" />
-      <label for="situation">Situation</label>
-      <select name="situation" id="situation-select">
-        <option value="">Select Situation</option>
-      </select>
-      <div id="title">
-        <form>
-          <label for="output-title">Output Title</label>
-          <input
-            size="100"
-            name="output-title"
-            id="output-title"
-            disabled="disabled"
-          />
-        </form>
-      </div>
-      <div id="table-wrap">
-        <table id="assetTable">
-          <thead></thead>
-          <tbody></tbody>
-        </table>
-        <div id="fill">
-          <button class="button" id="fillForm">Fill Form</button>
-        </div>
-        <textarea
-          id="description"
-          rows="10"
-          cols="100"
-          disabled="disabled"
-        ></textarea>
+
+<head>
+  <script type="text/javascript" src="jquery-3.6.0.min.js"></script>
+  <script type="text/javascript" src="popup.js" defer></script>
```

Comment:
By convention, script tags normally go in the head tag, so I just moved them up here. In this case, it needed 'defer' to wait for page to finish loading (which I assume is the reason they were placed at the bottom of the page).

Sorry for reformatting the rest of the html.